argon2.yaml

```diff
@@ -20,9 +20,15 @@ pipeline:
       tag: ${{package.version}}
       expected-commit: 62358ba2123abd17fccf2a108a301d4b52c01a7c
 
+  - uses: patch
+    with:
+      # The patch source is https://github.com/NixOS/nixpkgs/blob/e61356b9863f71e7b965f9b8ac3820e7d71af1a0/pkgs/development/libraries/libargon2/default.nix#L21
+      # It was merged upstream but a release hasn't been made yet.
+      patches: fix-cross-compilation.patch
+
   - runs: |
-      make -j$(nproc) OPTTARGET=none ARGON2_VERSION=${{package.version}}
-      make install DESTDIR=${{targets.destdir}} LIBRARY_REL=lib
+      make -j$(nproc) OPTTARGET=none ARGON2_VERSION=${{package.version}} LIBRARY_REL=lib PKGCONFIG_REL=lib
+      make OPTTARGET=none ARGON2_VERSION=${{package.version}} DESTDIR=${{targets.destdir}} LIBRARY_REL=lib PKGCONFIG_REL=lib install
```

Comment:
I'm not sure these changes are needed?  If we don't know what they do let's revert these and keep with the originals?